src/test/java/LoginPageTest.java

```diff
@@ -60,6 +60,60 @@ public void loginPage_LoginFormSection_HasIntroductionAndLoginFormCorrectlyDispl
         softAssertions.assertAll();
     }
 
+    @Test
+    public void loginPage_HeaderFooterNavigationDemoCustomer_AreSectionsCorrectlyDisplayed() {
+        //given
+        LoginPage loginPage = new LoginPage(driver);
+
+        //when
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(loginPage.isPageHeaderDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isPageNavigationBarDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isPageFooterDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isNewCustomerContainerDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isTryDemoCustomerAccessSectionDisplayed()).isTrue();
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void loginPage_NewCustomerSection_HasTitleAndTextAndButtonCorrectlyDisplayed() {
+        //given
+        LoginPage loginPage = new LoginPage(driver);
+
+        //when
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(loginPage.isNewCustomersTitleDisplayed()).isTrue();
```

Comment:
The same here: there is a missing another assertion - Before checking the presence of the title the presence of the whole section should be checked.